client/control/components/ActionForm.js

```diff
@@ -3,15 +3,15 @@ import { reduxForm } from 'redux-form';
 import HeartbeatForm, { HeartbeatFormFields } from './action_forms/HeartbeatForm.js';
 import ConsoleLogForm, { ConsoleLogFormFields } from './action_forms/ConsoleLogForm.js';
 
-function ActionForm({ fields, ChildForm }) {
+export function DisconnectedActionForm({ fields, ChildForm }) {
```

Comment:
The reason I renamed these components is because there is a new rule in airbnb that disallows importing the default export as a name of the other export. As an example, if you have a module

```js
export const FOO = 0;

export default function() { return FOO }
```

then it is a lint error to, in another file, `import FOO from './foo.js'`. It is confusing to import something by the name `FOO` that isn't `FOO`. I think it matters less in our case, but I tried to not change too many lint rules.